container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java

```diff
@@ -488,7 +488,8 @@ private void renderHitContents(Hit hit) throws IOException {
         if (id != null)
             generator.writeStringField(ID, id);
 
-        generator.writeNumberField(RELEVANCE, hit.getRelevance().getScore());
+        generator.writeFieldName(RELEVANCE);
+        generator.writeNumber(hit.getRelevance().toString());
```

Comment:
how will this interact with custom renderers?
do we expect them to use toString or getScore?

Reply:
(as in maybe we also need to change the XML renderer to unbreak the system tests)